lib/Docs/RST/Logger.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Docs\RST;
+
+use Psr\Log\AbstractLogger;
+use Stringable;
+
+use function date;
+use function sprintf;
+
+use const PHP_EOL;
+
+class Logger extends AbstractLogger
+{
+    /** @param mixed[] $context */
+    public function log(mixed $level, Stringable|string $message, array $context = []): void
+    {
+        echo sprintf('[%s] %s: %s', date('Y-m-d H:i:s'), $level, $message) . PHP_EOL;
```

Comment:
Yeah... A logger but what should we do? Will anyone ever look at this?

Reply:
Sorry, where is that logger used? Why is it necessary?

Reply:
I assume it appears in the console output when the build is run.

Reply:
yes this does the same. we could attach a normal PSR logger

Reply:
This should be sufficient for this PR. A logger can be added later.